src/test/java/com/twilio/rest/supersim/v1/SimTest.java

```diff
@@ -136,6 +136,18 @@ public void testUpdateFleetWithUniqueNameResponse() {
         Sim.updater("HSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX").update();
     }
 
+    @Test
```

Comment:
New tests being added? Think you need to rebase and force push. Let me know need help with this.